docker-compose.override.yml.dist

```diff
@@ -0,0 +1,6 @@
+services:
+  prestashop-git:
+    environment:
+      #BLACKFIRE_ENABLE: 1
+      #BLACKFIRE_SERVER_ID: "your_server_id"
+      #BLACKFIRE_SERVER_TOKEN: "your_server_token"
```

Comment:
```suggestion
      BLACKFIRE_ENABLE: 1
      BLACKFIRE_SERVER_ID: "your_server_id"
      BLACKFIRE_SERVER_TOKEN: "your_server_token"
```

It's already in a dist file, no need to comment since you have to remove the extension for this to be taken into account

Reply:
The file must be a model allowing a basic config, free for anyone wanting to have their own override to remove unwanted services, like phpmyadmin on develop https://github.com/PrestaShop/PrestaShop/blob/8457cb32fe3badcb15993f18eedd318e6dbbc33f /docker-compose.override.yml.dist#L12

Reply:
yes, but the uncommented values here are values that won't install additional tools in the containers, they configure things that will be installed no matter what

I find it safer to comment the parts that will add additional stuff

that being said I might switch to a blackfire image

Reply:
it's true that we don't comment phpmyadmin in this file on develop though